Client/Frontend/AuthenticationManager/AuthenticationSettingsViewController.swift

```diff
@@ -10,7 +10,7 @@ import LocalAuthentication
 private let logger = Logger.browserLogger
 
 private func presentNavAsFormSheet(_ presented: UINavigationController, presenter: UINavigationController?, settings: AuthenticationSettingsViewController?) {
-    presented.modalPresentationStyle = .formSheet
+    presented.modalPresentationStyle = UIDevice.current.isPhone ? .fullScreen : .formSheet
```

Comment:
```suggestion
    presented.modalPresentationStyle = .automatic
```

Same as before. The old style is deprecated.